c++/src/kj/exception.c++

```diff
@@ -714,6 +714,27 @@ retry:
   return filename;
 }
 
+void resetSignalHandlers() {
```

Comment:
As seen in the CI failures, this function doesn't compile on Windows.

Reply:
Ugh... Sorry I missed it since there are few failing build on master.
Added ifdefs around signal stuff (still keeping `set_terminate(null)` on Windows)

Reply:
Maybe we should name it `resetCrashHandlers()` then since "signals" are a Unix thing?

I fixed MinGW last week so there should be no failing quick tests on master.